app/controllers/books_controller.rb

```diff
@@ -5,7 +5,7 @@ class BooksController < ApplicationController
 
   # GET /books or /books.json
   def index
-    @books = Book.all
+    @books = Book.all.order(title: :asc).page(params[:page])
```

Comment:
```suggestion
    @books = Book.order(title: :asc).page(params[:page])
```

orderやwhereを付けるとallは省略可能です。
あとtitleは日本語が含まれる可能性があるので、orderとして使うカラムには不向きかもしれません。
参考 https://blog.jnito.com/entry/2014/12/03/185146

あと、同名の本が含まれる可能性もあるので、ユニーク性も保証できなさそうです。